Updated documentation for namespace scoped entity alias creation (VAULT-48944) (#3135)
hashicorp/web-unified-docs: content/vault/v2.x/content/api-docs/secret/identity/entity-alias.mdx

  set `issuer`. If you omit `mount_accessor`, Vault automatically generates a
  synthetic value for aliases bound to an OAuth resource-server configuration in
  the format `oauth-resource-server_<namespace_id>_<config_id>`, where `<config_id>`
  is the UUID of the matching configuration. You can also use a namespace path

>   in place of `<namespace_id>`: `oauth-resource-server_<namespace_path>_<config_id>`.

Reviewer: Was thinking about a way to make it a little more clear which formats are acceptable when specifying the mount accessor with an OAuth config:

```suggestion
  is the UUID of the matching configuration. You may also specify the mount 
  accessor in the format `oauth-resource-server_<namespace_id>_<config_id>` or
  `oauth-resource-server_<namespace_path>_<config_id>`.
```

Author: Thanks, this makes the intent much clearer.

>   **We recommend omitting `mount_accessor` and letting Vault generate this value automatically.**

Reviewer: Suggest clarifying when to let Vault generate the value.

```suggestion
  **We recommend omitting `mount_accessor` and letting Vault generate this value
  automatically when using [Vault as an OAuth resource server](/vault/docs/ai/oauth-server).**
```

Author: Good catch, it does help clarify the scenario and use case.

  set `issuer`. If you omit `mount_accessor`, Vault automatically generates a
  synthetic value for aliases bound to an OAuth resource-server configuration in
  the format `oauth-resource-server_<namespace_id>_<config_id>`, where `<config_id>`
  is the UUID of the matching configuration. You may also specify the mount 
  accessor in the format `oauth-resource-server_<namespace_id>_<config_id>` or
  `oauth-resource-server_<namespace_path>_<config_id>`.

  **We recommend omitting `mount_accessor` and letting Vault generate this value

>   automatically when using [Vault as an OAuth resource server](/vault/docs/ai/oauth-server).**

Reviewer 2: ```suggestion
<Tip>

  When using Vault as an [Vault as an OAuth resource server](/vault/docs/ai/oauth-server),
  we recommend omitting `mount_accessor` so Vault automatically generates the
  accessor value.

</Tip>
```
